oak-run/src/main/java/org/apache/jackrabbit/oak/run/DataStoreCopyCommand.java

```diff
@@ -83,6 +84,7 @@ public void execute(String... args) throws Exception {
                     item.source += "?" + sasToken;
                 }
                 item.destination = getDestinationFromId(id);
+                item.checksum = id.replaceAll("-", "");
```

Comment:
@amit-jain to get the expected checksum, I just remove the `-` from the blob id. Is there some function I can reuse to safely get the checksum from a blob identifier?